docs/anchoring-assets/anchor-platform/sep24/faq.mdx

```diff
@@ -32,7 +32,8 @@ You should use the `sub` field of the JWT token. For custodial wallets, this val
 
 ### How to identify the wallet?
 
-You should use a combination of `sub` and `data.client_domain` fields of the JWT token. For custodial wallets, the `sub` value will be in the format `account:memo`. Use the account to identify the wallet. For noncustodial wallets, use the `data.client_domain` field. Note that the wallet must provide the `client_domain` beforehand as a part of SEP-10 authentication.
+giYou should use a combination of `sub` and `data.client_domain` fields of the JWT token. For custodial wallets, the `sub` value will be in the format `account:memo`. Use the account to identify the wallet. For noncustodial wallets, use the `data.client_domain` field. Note that the wallet must provide the `client_domain` beforehand as a part of SEP-10 authentication. If [clients] configuration is present, the `data.client_name` field will also be included in the JWT token, which can be used to identify the wallet.
```

Comment:
This documentation is outdated. Our current recommendation is just to use clients